### ISSUE TITLE
Installs shared library files with mismatching version numbers

### ISSUE THREAD

**Issue by yurivict** [OpenImageDenoise/oidn#87]
The version oidn-1.2.3 installs these files:
```
lib/libOpenImageDenoise.so
lib/libOpenImageDenoise.so.0
lib/libOpenImageDenoise.so.1.2.3
```

```0``` should match the first digit in ```1.2.3```, otherwise some tools from toolchains might get confused.

**Comment by atafra** [comment]
The reason why the `SOVERSION` is still is 0 is that the `0.x.x` API is still forward compatible with `1.x.x`. Could you give a concrete example for a toolchain issue? Thanks!

**Comment by atafra** [comment]
Fixed in `v1.3.0`.
